lib/ramble/ramble/workspace/workspace.py

```diff
@@ -817,12 +820,18 @@ def all_experiments(self, workload):
 
         experiments = workload[experiment_namespace]
         for experiment, contents in experiments.items():
-            experiment_vars = None
+
+            # FIXME: what is the cleanest way to do this?
+            import ruamel
```

Comment:
I think we should use `spack.util.spack_yaml` instead of ruamel.

Reply:
OK great, that's what I had but it wasn't clear to me that it would resolve to the same `CommentedDict`, changed!